lib/auth/user_agent.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Auth.UserAgent do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Auth.Repo
+
+  schema "user_agents" do
+    field :user_agent, :string
```

Comment:
I typically would not call the main field of a schema an almost _identical_ name to the schema.
Think about writing a query: `SELECT user_agents.user_agent` the `IDE` will confuse the two and autosuggest/complete `user_agent` (_singular_) to `user_agents` plural.
I think we would be better off naming the field `name` or `agent`. 💭